docs/security-considerations.rst

```diff
@@ -65,6 +65,7 @@ To give an example, the following code contains a bug (it is just a snippet and
         mapping(address => uint) shares;
         /// Withdraw your share.
         function withdraw() public {
+        // This will report a warning (deprecation)
```

Comment:
I have left some of these comments because the text mentions `send`/`transfer` in the description or explanation of the examples. This should be rewritten to avoid that.

Reply:
no all good!